modules/odc-deploying-java-applications.adoc

```diff
@@ -26,7 +26,7 @@ Use the following instructions to upload a JAR file in the *Topology* view to de
 
 . In the *Name* field, enter a unique component name to name the associated resources.
 
-. In the *Resources* field, choose the resource type for your application.
+. Optional: You can select a different resource type from the default resource type if needed from the *Advanced options* -> *Resource type* drop-down list. 
```

Comment:
Drop the "if needed". The "Optional:" tag at the start of the sentence implies that this instruction is applicable only if needed for a specific use case.

Suggested revision:

Optional: To select a different resource type from the list of default resource types, use the **Advanced options** -> **Resource type** drop-down list.

Reply:
To maintain consistency, I would like to go with the second suggested version as follows:

Optional: Using the *Advanced options* -> *Resource type* drop-down list, select a different resource type from the list of default resource types. 

Cc @divyanshiGupta @sanketpathak